lexibank_chenhmongmien.py

```diff
@@ -34,90 +33,63 @@ class Dataset(NonSplittingDataset):
     concept_class = HConcept
     language_class = HLanguage
 
-    def cmd_download(self, **kw):
-        wp = requests.get(
-            "https://en.wiktionary.org/wiki/Appendix:Hmong-Mien_comparative_vocabulary_list"
-        )
-        soup = BeautifulSoup(wp.content, "html.parser")
-
-        language_table_header, language_table = [], []
-        languages = soup.findAll("table", {"class": "wikitable sortable"})[0]
-        for lh in languages.findAll("th"):
-            language_table_header.append(lh.get_text().rstrip("\n"))
-
-        for r in languages.findAll("tr"):
-            temp = []
-            for cell in r.findAll("td"):
-                temp.append(cell.get_text().rstrip("\n"))
-            language_table.append(temp)
-
-        language_table = [x for x in language_table if x != []]
+    def cmd_download(self, args):
+        with self.raw_dir.temp_download(
```

Comment:
`DataDir` has some support for downloading data. Not necessarily simpler in this case, but we don't need `requests`.

Reply:
given the dramas around `requests` it would be good idea to not use it if we can (or at least abstract it out into `temp_download`)

Reply:
@MacyL, please look into that.